lib/ansible/plugins/callback/default.py

```diff
@@ -16,8 +16,27 @@
       - default_callback
     requirements:
       - set as stdout in configuration
+    options:
+      check_mode_markers:
+        name: Show markers when running on check mode
+        description:
+        - "Toggle to control displaying markers when running on check mode. The markers are 'DRY RUN'
+        at the beggining and ending of playbooks execution (when calling ansible-playbook --check)
+        and 'CHECK MODE' as a suffix at every task that is run in 'check' mode."
+        type: bool
+        default: no
+        version_added: 2.9
+        env:
+          - name: ANSIBLE_CHECK_MODE_MARKERS
+        ini:
+          - key: check_mode_markers
+            section: defaults
 '''
 
+# NOTE: check_mode_markers functionality is also implemented in the following derived plugins:
```

Comment:
The code is implemented there but with the options only existing in the default callback plugin, there is no way to enable it with other callback plugins. 

I would suggest removing this note and moving the `check_mode_markers` documentation to the default doc fragment (`lib/ansible/plugins/doc_fragments/default_callback.py`). This will allow the setting to be controlled from all those plugins as well as update the documentation for all those plugins.

Reply:
Hello, please have a look at [comment](https://github.com/ansible/ansible/pull/49432#issuecomment-485834431) above for clarification, regarding a discussion we had with bcoca about this.

In short, there are plugins which use the default callback's doc fragment but do not inherit from it (like others).

So in the present situation of this patch, where documentation is in the default plugin, there is an undocumented feature for the three mentioned plugins. 

On the other hand if I move the documentation to the default callback's doc fragment, there will be 2 plugins that will have their documentation mention the feature but in fact do not implement it.

Reply:
Ok. I did not see that previous discussion.